docs/lib/datastore/fragments/flutter/conflict.md

```diff
@@ -0,0 +1,3 @@
+```
+Coming Soon
```

Comment:
This is not descriptive. Proposed edit:
Custom conflict resolution configuration is not available just yet for Flutter. We are actively working on this. Track our progress with this issue: [...]

Reply:
Added.